app/models/pregnancy.rb

```diff
@@ -50,4 +51,22 @@ class Pregnancy
   #   end
   #   false
   # end
+  
+  private
+  
+  def updating_pledge_sent?
+    pledge_sent == true
+  end
+
+  def check_other_validations?
+    if dcaf_soft_pledge.blank?
```

Comment:
If these aren't set, we want them to all spit out, yeah? What are your thoughts on turning these into a bunch of one-liners? e.g.:

```
errors.add(:pledge_sent, 'DCAF soft pledge field cannot be blank') if dcaf_soft_pledge.blank?
# ... and so on
```